qiita_db/test/test_util.py

```diff
@@ -575,7 +575,10 @@ def test_move_upload_files_to_trash(self):
 
         # create file to move to trash
         fid, folder = get_mountpoint("uploads")[0]
-        open(join(folder, '1', test_filename), 'w').write('test')
+        test_fp = join(folder, '1', test_filename)
+        open(test_fp, 'w').write('test')
```

Comment:
I know the garbage collector is supposed to get rid of these unclosed file handles, but I rather have this handle be closed explicitly (using with open maybe?).